src/coreclr/nativeaot/Runtime/unix/HardwareExceptions.cpp

```diff
@@ -587,6 +594,27 @@ bool InitializeHardwareExceptionHandling()
         return false;
     }
 
+#if defined(HOST_OSX)
+	// LLDB installs task-wide Mach exception handlers. XNU dispatches Mach
```

Comment:
Does this problem exists in regular CoreCLR as well - should the same fix be made there?

Reply:
It doesn't. We don't use signals for sigsegv / arithmetic errors there and we use a mach exception handler.